gradle.properties

```diff
@@ -4,7 +4,7 @@ android.useAndroidX=true
 
 # POM
 GROUP = com.dropbox.kaiken
-VERSION_NAME=2.0.0
+VERSION_NAME=2.0.1-SNAPSHOT
```

Comment:
bad that this wasn't set earlier, let's add to release script if not there

Reply:
It's there, I just forgot to do it.